uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestCalendarFeedDataTest.php

```diff
@@ -82,26 +82,30 @@ public function testGetWillReturnDataForFeedConfigLocationAndDepartmentContacts(
     HRJobRolesFabricator::fabricate([
       'job_contract_id' => $contract1['id'],
       'department' => $department1['value'],
+      'start_date' => date('Y-m-d H:i:s'),
     ]);
 
     //contact2 assigned to location1
     HRJobRolesFabricator::fabricate([
       'job_contract_id' => $contract2['id'],
-      'location' => $location1['value']
+      'location' => $location1['value'],
+      'start_date' => date('Y-m-d H:i:s')
     ]);
 
     //contact3 assigned to location1 and department1
     HRJobRolesFabricator::fabricate([
       'job_contract_id' => $contract3['id'],
       'location' => $location1['value'],
-      'department' => $department1['value']
+      'department' => $department1['value'],
+      'start_date' => date('Y-m-d H:i:s')
     ]);
 
     //contact4 assigned to both location2 and department2
     HRJobRolesFabricator::fabricate([
       'job_contract_id' => $contract4['id'],
       'location' => $location2['value'],
-      'department' => $department2['value']
+      'department' => $department2['value'],
+      'start_date' => date('Y-m-d H:i:s')
```

Comment:
No need for these changes once the start date is set as default in the fabricator.